api/v1beta1/osccluster_validation.go

```diff
@@ -61,6 +61,16 @@ func ValidateCidr(cidr string) (string, error) {
 	return cidr, nil
 }
 
+// ValidateSubregionName check that subregionName is a valid az format
```

Comment:
→ api/v1beta1/oscmachine_validation.go:183:6: ValidateSubregionName redeclared in this block
       → 	api/v1beta1/osccluster_validation.go:65:6: other declaration of ValidateSubregionName
       
       You can not have the same name.
       Please keep only ValidateSubregionName in oscmachine_validation because it is the same.